modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartDioClientCodegen.java

```diff
@@ -40,7 +40,7 @@
 
     private static final String CLIENT_NAME = "clientName";
 
-    private boolean nullableFields = true;
```

Comment:
What do you think to move this to dart generator?

Reply:
I would rather deprecate this and get rid of it in 6.0 -  we should not promote this going forward in with Dart/NNBD. I think this was more of a workaround for built_value. Maybe @josh-burton can say more to the original motivation.